games/o.yaml

```diff
@@ -986,6 +986,19 @@
   video:
     youtube: Xfv_UrDq-As
 
+- name: OpenNotrium
+  lang: C++
+  license:
+  - Custom
```

Comment:
```suggestion
```
I think this is not necessary; Ville's terms basically specify what kind of licenses he wants, of which GPL3 fits. So the project OpenNotrium is only available under GPL3, as Ville's terms only apply between him and OpenNotrium, and not OpenNotrium's users.

Reply:
I am not a lawyer, but I think Ville's clauses have a certain impact on the users of OpenNotrium too: it seems, for example, prohibited to package and sell OpenNotrium (say, on a console platform), or sell your own fork of OpenNotrium, as "Any releases shall be free for the end user". I think GPL would want users to have those rights?

However, the spirit is certainly to stay as close as possible to the GPL. I'm also not sure how important this potential (maybe even pedantic) issue is in the context of the opengaming repo.

Thoughts?

Reply:
The GPL3 license also prohibit the selling of the product.

I think the GPL3 plus Custom license is correct, because Ville has the rights to Notrium. If I read correctly, only the OpenNotrium project has rights to use the original code and assetts from Notrium, the added codes must be licensed under GPL3.